tracks/bash/exercises/proverb/mentoring.md

```diff
@@ -0,0 +1,44 @@
+### Reasonable Solutions
+```bash
+#!/usr/bin/env bash
+
+main() {
+  if [ $# -eq 0 ]; then
+    echo ""
+    exit
+  fi
+  for ((currentNounIndex = 1, nextNounIndex = 2; currentNounIndex < $#; currentNounIndex++, nextNounIndex++)); do
+    proverb+=$"For want of a ${!currentNounIndex} the "${!nextNounIndex}" was lost.\n"
+  done
+  proverb+="And all for the want of a $1."
+  echo -e "$proverb"
+}
+
+main "$@"
+```
+
+### Common suggestions
+- use "$#" to get number of arguments
```

Comment:
Should use code marks to designate what should be code. Is the `$#` what is meant here?  Or is it `"$#"`?  The marks can help make that clearer.

Reply:
understood I will change it to show `"$#"`, thanks 🎉